src/Checkbox/Checkbox.web.js

```diff
@@ -33,16 +20,15 @@ export default function Checkbox({
   name,
   info,
   onChange,
-}: Props) {
-  const labelStyle = {
+}: CheckboxProps) {
+  const styleLabel = {
```

Comment:
```suggestion
  const labelStyle = {
```
I think we prefer this way of writing variables, but I am curious to know your reason for that change. Do you think it could be problematic later on?

Reply:
No, you are right. It makes more sense this way. I probably just used this pattern because of switching from `react-native` styling which is usually something like `styles.sublabel`.